samples/docker_aks/Dockerfile

```diff
@@ -1,7 +1,17 @@
+FROM ubuntu:20.04 as builder
+
+RUN apt-get update && apt-get install -y wget gcc
```

Comment:
nit: good practice to apt clean and remove temp files afterwards -- this prevents unnecessary files in the layer
curl may also be an option to use so you don't have to install wget

Reply:
Would they not get deleted automatically since this is the first layer?

Reply:
Yup, it would still be discarded anyways in the final image so it's not important to minimize this intermediate layer.